src/test/java/bbd/jportal2/test/FreeMarkerTest.java

```diff
@@ -82,31 +85,43 @@ public void testFreeMarkerMultipleSubDirMultipleFiles() throws Exception {
         String nubDir="";
         Database db = JPortal.run(siFile.toString(), nubDir);
         File outputDirectory = new File(outputDir.toString());
-        (new FreeMarker()).generateTemplate(db, TEMPLATE_DIR, nameOfTest, outputDirectory);
+
+        Table tab = null;
+        for (Table t : db.tables) {
+            if (t.name.equals("ExampleTable")) {
+                tab = t;
+                break;
+            }
+        }
+
+        (new FreeMarker()).generateTemplate(db, tab, TEMPLATE_DIR, nameOfTest, outputDirectory);
 
         //Test output - dir1/file1.txt
         List<String> lines = Files.readAllLines(Paths.get(outputDirectory.toString(), "dir1/file1.txt"));
-        assertEquals("Database Name: jportal_example_db" ,lines.get(0));
-        assertEquals("Field Name: ID"           ,lines.get(1));
-        assertEquals("Static: 1"                ,lines.get(2));
+        assertEquals("Database Name: jportal_example_db",lines.get(0));
```

Comment:
Side note:
We should find a better way to compare the output to expected. I think we should have a directory with the expected output files, and then do a file-based compare rather than a line-by-line string compare. Not important for now, but something to add.

Reply:
I've created an Issue for this so we can make some enhancements. 

https://github.com/SI-Gen/jportal2/issues/69